ale/drivers/base.py

```diff
@@ -216,6 +216,25 @@ class Driver():
     _file : str
             Reference to file path to be used by mixins for opening.
     """
+
+    def __enter__(self):
+        """
+        Called when the context is created. This is used
+        to get the kernels furnished.
+        """
+        if self.metakernel:
+            spice.furnsh(self.metakernel)
+            print("Furnished")
+        return self
```

Comment:
Why are the functions for furnishing and unfurnishing kernels being moved into the pds3 driver mixin from the spice mixin?

Reply:
Was originally a hacky way to get ISIS3 label + naif spice kernel. Should be reverted.